tests/untl_structure_test.py

```diff
@@ -298,7 +299,7 @@ def test_UNTLElement_record_length():
     # "{'meta': [{'qualifier': 'ark', 'content': 'fake'}], 'collection': [{'content': 'Colección'}]}"  # noqa 
     # We are getting that it is length 97 for:
     # "{'meta': [{'content': 'fake', 'qualifier': 'ark'}], 'collection': [{'content': u'Colecci\xf3n'}]}"  # noqa
```

Comment:
@ldko, can we remove this and the other comments like it now that the numbers should match up with the expected counts (apart from being off by one here)?

Reply:
oh I just double checked, the 94 I put in the comment should have been a 93, so yes, I think we can remove the "NOTE" comments in `test_UNTLElement_record_length` and `test_UNTLElement_record_content_length`.